docs/pages/deployment/cli/commands/nuts_network_get.rst

```diff
@@ -0,0 +1,39 @@
+.. _nuts_network_get:
+
+nuts network get
+----------------
+
+Gets a transaction from the network
+
+Synopsis
+~~~~~~~~
+
+
+Gets a transaction from the network
+
+::
+
+  nuts network get [ref] [flags]
+
+Options
+~~~~~~~
+
+::
+
+  -h, --help   help for get
+
+Options inherited from parent commands
```

Comment:
- Inconsistent use of `[flags]` and `options` is confusing.
- Not all options/flags/inputs are listed, but this is probably more a documentation thing.

Reply:
out of scope